api/agent/agent.go

```diff
@@ -109,7 +110,12 @@ type agent struct {
 	shutWg   *common.WaitGroup
 	shutonce sync.Once
 
+	// TODO(reed): shoot this fucking thing
```

Comment:
Yes

Reply:
let's not add profanity to the code base.